src/main/kotlin/parts/wisdom/arcraider/visualization/ARCWindow.kt

```diff
@@ -88,9 +94,35 @@ class ARCWindow(title: String, task: Task) : JFrame() {
         }
     }
 
-    private fun refresh(task: Task) {
+    private fun renderGame(task : Task, game : Game) {
+        val input = game.getInputGrid()
+        val output = game.getOutputGrid()
+        val curGrid = game.getCurrentGrid()
+
+        content.apply {
+            add(Box.createHorizontalBox().apply {
+                add(Box.createRigidArea(Dimension(GAP_SIZE_PX, 0)))
+                add(GridPane(input, false))
```

Comment:
Raises the obvious question of whether it's good or bad that `renderGame` and `renderTask` are different methods with some redundancy of code. I see pros and cons depending on how this evolves. I probably would have added `if` statements to `renderTask` instead, but I'm ok either way.